client/src/components/GroceryList/GroceryListItem.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import styled from "styled-components/macro";
+import IconButton from "../BottomNav/IconButton";
+import Checkbox from "../../assets/checkbox.svg";
+import CheckboxChecked from "../../assets/checkboxChecked.svg";
+import PropTypes from "prop-types";
+
+const ListItem = styled.p`
+  display: inline-flex;
+  margin: 0.4rem 0;
+  button {
+    margin: 1px 5px 0 0;
```

Comment:
Maybe stick to the same unit for margin/padding/etc. for more consistency

Reply:
thanks :)
I wanted to fix the button with the px...the generel item spacing was supposed to be variable :)